kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -45,7 +45,7 @@
         <h3 class="title" dir="auto">
           <TextTruncatorCss
             :text="content.title"
-            :maxLines="3"
+            :maxLines="1"
```

Comment:
Is it necessary for this just to be "1"? When we had looked at it together, it seemed like having the max lines set to 2 would be acceptable and not cause a conflict. I'm worried that having only one line might result in not enough text being displayed for some longer titles. If you have a screenshot of an example that demonstrates why "1" is better, that would be helpful to include in the PR!

Reply:
Hi Marcella! I have tried setting it to 2. But I realised that setting it to one is actually better. 
[screen-capture (3).webm](https://user-images.githubusercontent.com/103313919/204302857-132c4ef4-5f3e-41a5-b37b-b58cc566ba7d.webm)

Please don't merge it yet today as I explore more alternatives.